docs/modules/ROOT/pages/how-to/day2ops/remove-organisation.adoc

```diff
@@ -0,0 +1,17 @@
+= Remove an organisation
+
+== Prerequisites
+
+* All namespaces on every APPUiO Cloud zone, belonging to that organisation, is removed.
+* https://kb.vshn.ch/appuio-cloud/how-to/day2ops/connect-control-api.html[Connect to control api]
+
+
+== Remove organisation
+
+
+[source,bash]
+----
+org=<NAME_OF_THE_ORG>
+
+kubectl delete organization $org
```

Comment:
```suggestion
kubectl delete organization <name>
```

I kind of hate that notation. Actually, I find it dangerous. There can be a risk of the variable set to a wrong value when mixing up terminals.

Reply:
I agree, but thats how the docu is built :D